#! /bin/sh -e

# DP: Modified

if [ $# -ne 1 ]; then
    echo >&2 "$0: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -f -p0 < $0;;
    -unpatch) patch -f -R -p0 < $0;;
    *)
        echo >&2 "$0: script expects -patch|-unpatch as argument"
        exit 1
esac
exit 0

--- contrib/spambayes.el.orig	2003-11-17 22:47:47.000000000 +0100
+++ contrib/spambayes.el	2004-12-21 15:30:25.000000000 +0100
@@ -84,7 +84,7 @@
 ;;      "X-Spambayes-Classification:"))
 
 
-(defvar spambayes-filter-program "/usr/local/bin/sb_filter.py"
+(defvar spambayes-filter-program "/usr/bin/sb_filter.py"
   "Path to the sb_filter program.")
 
 
